### ISSUE TITLE
[quick_actions] iOS example: get selected QuickAction not working.

### ISSUE THREAD

**Issue by jerryzhoujw** [flutter/flutter#41587]
## Steps to Reproduce

1. Run the example in [quick_action] with iOS device.
2. Go to Home Screen of device by minimize example app.
3. Long click example app to show the QuickActions.
4. Select one of the QuickActions.
5. After app load, the `MethodCallHandler` not get called. The navigation bar's title still be `no action set`. it should be `Action one` or `Action two` base on QuickActions selection.

## flutter doctor -v
```
[✓] Flutter (Channel beta, v1.9.1+hotfix.4, on Mac OS X 10.14.6 18G95, locale en-CN)
    • Flutter version 1.9.1+hotfix.4 at /Users/jerryzhou/Documents/code/flutter/sdk/flutter
    • Framework revision cc949a8e8b (2 days ago), 2019-09-27 15:04:59 -0700
    • Engine revision b863200c37
    • Dart version 2.5.0

[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/jerryzhou/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.0)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.0, Build version 11A420a
    • CocoaPods version 1.7.5

[✓] Android Studio (version 3.4)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 34.0.2
    • Dart plugin version 183.5901
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)

[✗] Cannot determine if IntelliJ is installed
    ✗ Directory listing failed

[✓] VS Code (version 1.38.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.4.1

[✓] Connected device (1 available)
    • iPhone • 1daca728d75535c21ced151fc7829191064d0e00 • ios • iOS 13.1.1
```

**Comment by augustinreille** [comment]
Any updates? Still not working in current version.

**Comment by jerryzhoujw** [comment]
@augustinreille  more attentions to this issue maybe cause some fix, otherwise I don't this will been fix in this year.

**Comment by yuvals38** [comment]
Hi
Any updates on this fix for ios
I am having trouble trying to figure out what code im missing in swift for the flutter action to take place

**Comment by scriptam** [comment]
Seems like a duplicate of https://github.com/flutter/flutter/issues/13634

See solution for iOS Swift - https://github.com/flutter/flutter/issues/13634#issuecomment-596128168

Add the following to `ios/Runner/AppDelegate.swift`

```
@available(iOS 9.0, *)
override func application(_ application: UIApplication, performActionFor shortcutItem: UIApplicationShortcutItem, completionHandler: @escaping (Bool) -> Void) {
    let controller : FlutterViewController = window?.rootViewController as! FlutterViewController
    
    let channel = FlutterMethodChannel(name: "plugins.flutter.io/quick_actions", binaryMessenger: controller.binaryMessenger)
    channel.invokeMethod("launch", arguments: shortcutItem.type)
    }
}
```

**Comment by iapicca** [comment]
Hi @jerryzhoujw 
as @scriptam pointed out (thank you) 
there's an open [issue](https://github.com/flutter/flutter/issues/13634) addressing the case you described.
Please follow up on that issue, I'm closing the current one as duplicate.
If you disagree, please write in the comments and I will reopen it.
Thank you
